fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/ContainerServiceImpl.java

```diff
@@ -63,7 +63,12 @@ public Container findOrCreate(final Session session, final String path) {
                 initializeNewObjectProperties(node);
             }
 
-            return new ContainerImpl(node);
+            final ContainerImpl container = new ContainerImpl(node);
```

Comment:
Are we only going to use `fedora:lastModified` on containers? not Binaries? With this ticket, we may run into the same issue with binaries:
https://jira.duraspace.org/browse/FCREPO-1665

Would it make sense to call `touch()` in `AbstractService.findOrCreateNode()`?